custom_components/xiaomi_cloud_map_extractor/common/image_handler.py

```diff
@@ -209,7 +274,7 @@ def draw_func(draw: ImageDraw):
             s = path.path[0].to_img(image.dimensions)
             for point in path.path[1:]:
                 e = point.to_img(image.dimensions)
-                draw.line([s.x * scale, s.y * scale, e.x * scale, e.y * scale], width=int(scale), fill=color)
+                draw.line([s.x * scale, s.y * scale, e.x * scale, e.y * scale], width=int(scale)*2, fill=color)
```

Comment:
Why?

Reply:
Ooops, this is just a variation, I prefer the path to be a little more visible. Just my preference, we can remove it or make it configurable as well.

Reply:
It shouldn't be a problem to make it configurable 👍